aten/src/ATen/native/cpu/ReduceOpsKernel.cpp

```diff
@@ -22,6 +22,15 @@ static void sum_kernel_impl(TensorIterator& iter) {
   });
 }
 
+static void mean_kernel_impl(TensorIterator& iter) {
+  AT_DISPATCH_ALL_TYPES(iter.type(), "sum", [&] {
+    binary_kernel_reduce_vec(
+      iter,
+      [=](scalar_t a, scalar_t b) -> scalar_t { return a + b; },
```

Comment:
Question: do we need to divide by the number of elements here? Also, the name just above passes "sum" to the macro

Reply:
I've read the code a bit, and I think, yes it should.

Reply:
Good catch. It's still doing the sum_out then divided by number of elements.

I throw the dummy mean_kernel_impl here at the beginning since TensorIterator was using the DISPATCH thing that requires a CPU implementation.

Totally forgot about this one since then. Let me remove this and clean it up.